Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/main.cpp

```diff
@@ -191,22 +191,23 @@ void UnregisterForActivation()
 
 void GetActivationInfo()
 {
-    AppActivationArguments args = AppInstance::GetCurrent().GetActivatedEventArgs();
+    AppActivationArguments args =
```

Comment:
Are these changes the same as the other PR?

Reply:
Yes.